torch/autograd/gradcheck.py

```diff
@@ -202,7 +205,8 @@ def gradcheck(
     raise_exception: bool = True,
     check_sparse_nnz: bool = False,
     nondet_tol: float = 0.0,
-    check_undefined_grad: bool = True
+    check_undefined_grad: bool = True,
+    check_grad_dtypes: bool = False
```

Comment:
@mruberry this new arg was not documented here. Is that on purpose on just an oversight?

Reply:
I don't recall offhand what we decided. This is probably an oversight? We may have thought to not document it at the time so we'd have more time to review the UX for this function later, though.

Reply:
Ok. Do you think it is stable enough now that we should document it? Or it is still fairly internal?

Reply:
Your call. I'm not planning any more changes to this function.